src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -4268,7 +4266,7 @@ private SymbolInfo GetNamedArgumentSymbolInfo(IdentifierNameSyntax identifierNam
                         default:
                             continue; // Definitely doesn't have parameters.
                     }
-                    ParameterSymbol param = FindNamedParameter(invocationSym.GetParameters(), argumentName);
+                    ParameterSymbol param = FindNamedParameter(invocationSym.GetSymbol().GetParameters(), argumentName);
                     if ((object)param != null)
                     {
                         symbols.Add(param);
```

Comment:
>param [](start = 36, length = 5)

Consider using `ArrayBuilder<ISymbol> symbols` and calling `GetPublicSymbol()` here.